tensorrt_llm/_torch/modules/fused_moe/quantization.py

```
@@ -1614,7 +1614,7 @@ def load_all_fp4_weight_scales_and_alphas(
             expert_idx = local_slot_id
 
             if not torch.allclose(w1_weight_scale_2, w3_weight_scale_2):
-                logger.warning(
+                tensorrt_llm.logger.warning(
```

Ruff (0.13.3)

Undefined name `tensorrt_llm`

(F821)